css/styles.css

```diff
@@ -8,21 +8,28 @@ html, body {
     padding: 0;
     font-family: Arial, Helvetica, sans-serif;
     text-align: center;
-}
-
-body {
-    background-color: #fafafa;
+    background-color: #333333;
+    color: #FFFFFF;
 }
 
 .container {
     width: 500px;
-    text-align: center
 }
 
-.container p {
+#path-sentence {
     visibility: hidden;
 }
 
 #nPaths {
     font-weight: bold;
 }
+
+.viz {
+    border: 1px #FFFFFF solid;
```

Comment:
These borders are temporary. I only added them so we could more easily start to see the layout of the page.